facia/conf/application.conf

```diff
@@ -1,6 +1,8 @@
 
 # Caching for humans.txt
 "assets.cache./public/humans.txt"="public, max-age=900"
+"assets.cache./public/security.txt"="public, max-age=0"
+"assets.cache./public/security.txt.asc"="public, max-age=0"
```

Comment:
We could probably cache these for the same duration as `humans.txt`?

Reply:
I was thinking about it to be honest because chances that they will change frequently is very very slim

Reply:
Let me make a PR for that then :)

Reply:
https://github.com/guardian/frontend/pull/22366